sdk/spring/azure-spring-boot-samples/azure-cloud-foundry-service-sample/pom.xml

```diff
@@ -21,8 +21,8 @@
   <dependencies>
     <dependency>
       <groupId>com.microsoft.azure</groupId>
-      <artifactId>azure-cosmosdb-spring-boot-starter</artifactId>
-      <version>2.3.3</version> <!-- {x-version-update;com.microsoft.azure:azure-cosmosdb-spring-boot-starter;dependency} -->
+      <artifactId>azure-cosmosdb-spring-boot-2-3-starter</artifactId>
+      <version>2.4.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:azure-cosmosdb-spring-boot-2-3-starter;current} -->
```

Comment:
Let's revert this change, our samples should reference the latest released version of our starter / dependencies.